lib/events/eventParser.js

```diff
@@ -120,7 +120,12 @@ EventParser._parseZoneGroupTopologyEvent = async function (body, device) {
     }
   }
   if (eventData.ZoneGroupState) {
-    eventData.Zones = eventData.ZoneGroupState.ZoneGroups.ZoneGroup.map(zone => { return new SonosGroup(zone) })
+    const zoneGroup = eventData.ZoneGroupState.ZoneGroups.ZoneGroup
+    if (Array.isArray(zoneGroup)) {
+      eventData.Zones = zoneGroup.map(zone => { return new SonosGroup(zone) })
+    } else {
+      eventData.Zones = new SonosGroup(zoneGroup)
```

Comment:
I suppose it ultimately depends on the handler, but is `eventData.Zones` normally expected to be an array? If so, perhaps this should be set to `[new SonosGroup(zoneGroup)]`.

Reply:
Good point. I'll change it to an array. Thanks for catching that.